README.md

```diff
@@ -55,6 +55,9 @@ driver:
 The `image_ref` and `flavor_ref` options can be specified as an exact id,
 an exact name, or as a regular expression matching the name of the image or flavor.
 
+All of Fog's `openstack` options (`openstack_domain_name`, `openstack_project_name`,
```

Comment:
Yeah this is fine. People looking for the options'll know what to look for. Thanks!